docs/tools/python-client-library/python-client-example.md

```diff
@@ -1,41 +1,113 @@
 ---
-title: Python client library usage example
+title: Python client examples
 layout: default
 parent: Python client library
 grand_parent: Tools
-nav_order: 2
+nav_order: 5
 ---
 
-# Example application using python client library
+# FeatureBase Python client examples
 
-In this example, FeatureBase python client library is used to bulk insert data into a demo table.
+These examples demonstrate how FeatureBase Python client classes work in practice.
 
 ## Before you begin
 
-{% include /cloud/cloud-before-begin.md %}, Or
-{% include /com-install/com-install-before-begin.md %}
-* [Learn How To Install python client library](/docs/tools/python-client-library/python-client-install)
+* [Install FeatureBase Python client library](/docs/tools/python-client-library/python-client-install)
+* [Learn how to connect to FeatureBase Cloud](/docs/tools/python-client-library/python-client-connect-cloud)
+* [Learn how to connect to FeatureBase Community](/docs/tools/python-client-library/python-client-connect-community)
+* [Learn how to run SQL queries against your database](/docs/tools/python-client-library/python-client-query)
 
-## Example python application
+{: .note}
+* Add or remove `#` characters to disable or enable the connection classes
 
-```python
+## Example one - SQL queries
+
+This example contains SQL queries to:
+* create a table
+* insert data
+* query the table
+
+Results are output using the Python `PRINT()` function
+
+```py
+# import the library
+import featurebase
+
+# FeatureBase Cloud client
+print("Connecting to FeatureBase Cloud...")
+c_client = featurebase.client(
```

Comment:
This can be `client = ` instead of `c_client`.  If this is renamed as suggested then this change should be replicated throughout the sample code. References to the client object is mixing the two versions of these names and that makes the sample code buggy and will error if user tries it.

Reply:
updated, thanks :)